src/Crypto/Lol/Cyclotomic/Tensor/CTensor.hs

```diff
@@ -283,7 +286,12 @@ instance (TElt CT (ZqBasic q i), Dispatch (ZqBasic q i)) => CRNS (ZqBasic q i) w
   divGWrapper f = f
   gaussWrapper = error "Cannot call gaussianDec for ZqBasic"
 
-instance (Storable a, Storable b, CRNS a, CRNS b, CRTrans a, CRTrans b) 
+instance (Storable a, Storable b, 
+          CRNS a, CRNS b, 
+          CRTrans a, CRTrans b, 
+          ZeroTestable a, ZeroTestable b, 
```

Comment:
Why all the new constraints (`ZT, ID, Random`) here?  They weren't needed before...